spec/features/admin/user_role_spec.rb

```diff
@@ -7,10 +7,12 @@
 
   describe '#index' do
     before { visit admin_user_roles_path }
-    it { expect(page_title).to have_content('ロール') }
-    it { expect(page).to have_content(user.name) }
-    it { expect(page).to have_content('作成する') }
-    it { expect(page).to have_css('.delete_link') }
+    it 'should open the index page' do
+    expect(page_title).to have_content('ロール')
```

Comment:
__[rubocop]__
* Use 2 (not 0) spaces for indentation. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-indentation)

<!-- Comment by [SideCI](https://sideci.com) -->